profiler/src/main/java/com/splunk/opentelemetry/profiler/context/SpanContextualizer.java

```diff
@@ -53,27 +54,27 @@ public void updateContext(RecordedEvent event) {
   }
 
   /** Parses thread info from the raw stack string and links it to a span (if available). */
-  public StackToSpanLinkage link(Instant time, String rawStack) {
+  public StackToSpanLinkage link(Instant time, String rawStack, RecordedEvent event) {
```

Comment:
these changes seem relatively unrelated to the export pipeline. Any reason this couldn't be split out into a separate PR?

Reply:
It would be a lot of work at this point because the PRs were already stacking up...it was a decent amount of work to get to this stage already.  🙏🏻